.github/workflows/golangci-lint.yml

```diff
@@ -17,5 +17,5 @@ jobs:
         uses: golangci/golangci-lint-action@v2.4.0
         with:
           version: v1.29
-          args: --timeout=5m
+          args: --timeout=5m --enable=gosec
```

Comment:
There is another PR with all the linters enabled https://github.com/ossf/scorecard/pull/184. Could you please remove this? The other one should take of gosec also.